scripts/scan.sh

```
@@ -1 +1,10 @@
-# TODO: write the script
+printf "\n====== SCANNING FOR CREDIT CARD NUMBERS ======\n"
+grep -rE --color=always '(\b[0-9]{4}[- ]?){3}[0-9]{4}\b' . --exclude-dir={.git} --line-number
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)